src/openms/source/ANALYSIS/MAPMATCHING/QTClusterFinder.cpp

```diff
@@ -337,7 +338,12 @@ namespace OpenMS
          it = elements.begin(); it != elements.end(); ++it)
     {
       feature.insert(it->first, it->second->getFeature());
+      if (it->second->getFeature().metaValueExists("dc_charge_adducts"))
```

Comment:
it->second->getFeature() is accessed a few times - introduce const & to point to it?